js/textInputModel.js

```diff
@@ -154,7 +154,7 @@ class TextInputModel extends QuestionModel {
       const userAnswer = item.userAnswer || '';
       const isCorrect = this.checkAnswerIsCorrect(answers, userAnswer);
```

Comment:
We probably don't need to call `this.checkAnswerIsCorrect` here as: 
```js
item._isCorrect = (item._answerIndex !== -1);
```
This is because `getAnswerIndex` effectively does everything `checkAnswerIsCorrect` does, but it returns an index instead of a boolean.